detection-rules/impersonation_microsoft_quarantine.yml

```diff
@@ -5,32 +5,34 @@ type: "rule"
 severity: "high"
 source: |
   type.inbound
-  and length(filter(attachments, .file_type not in~ ('png', 'jpeg', 'jpg', 'bmp', 'gif'))) == 0 
+  and all(attachments, .file_type in~ ('png', 'jpeg', 'jpg', 'bmp', 'gif'))
   and any(attachments,
-      any(file.explode(.),
-          any(ml.nlu_classifier(.scan.ocr.raw).intents, .name == "cred_theft" and .confidence != "low")
-          and any(ml.nlu_classifier(.scan.ocr.raw).entities, .name == "urgency")
-      )
-       and any(file.explode(.),
-          3 of (
-              any(.scan.strings.strings, strings.icontains(., "review")),
-              any(.scan.strings.strings, strings.icontains(., "release")),
-              any(.scan.strings.strings, strings.icontains(., "quarantine")),
-              any(.scan.strings.strings, strings.icontains(., "messages")),
-              any(ml.logo_detect(..).brands, strings.starts_with(.name, "Microsoft"))
```

Comment:
Nesting logo_detect inside `file.explode` was a little odd, so I refactored this so they are all at the same place. This still keeps the same meaning though! But I can back it out if it's confusing